### ISSUE TITLE
Possibility to change all textfields

### ISSUE THREAD

**Issue by henriklagergren** [akosidencio/react-cart-components#1]
I want to use your component but can only change some textfields. Since I'm writing in another language I want to be able to change all textfields ("Cart", "Quantity" and "Subtotal").

**Comment by akosidencio** [comment]
@henriklagergren will do some updates to make that possible

**Comment by akosidencio** [comment]
@henriklagergren currently the 'Checkout' and 'Add to Cart' label has been supported to change text.

**Comment by henriklagergren** [comment]
@akosidencio Yes I saw that, but to use it in another language I need to be able to also change 'Cart', 'Quantity', 'Subtotal' and the text when the cart is empty.
